php/admin/class-settings.php

```diff
@@ -77,21 +77,14 @@ public function register_settings() {
 	 * Render the Settings page.
 	 */
 	public function render_page() {
-		$tab = filter_input( INPUT_GET, 'tab', FILTER_SANITIZE_STRING );
-
-		ob_start();
-
-		$this->render_page_header();
-
-		switch ( $tab ) {
-			case 'license':
-			default:
-				include block_lab()->get_path() . 'php/views/license.php';
-				break;
-		}
-
-		$html = ob_get_clean();
-		echo '<div class="wrap block-lab-settings">' . $html . '</div>'; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+		?>
+		<div class="wrap block-lab-settings">
```

Comment:
Good move simplifying this. Obviously the previous solution is intended to provide a way of expanding the tabs to include additional ones in the future, but we can redo that when we need it.

Reply:
Thanks, I wasn't quite sure if this needed simplifying. Like you mentioned, we could revert the commit and use the previous `switch` when needed.